com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -53,9 +53,18 @@ public class NetworkConfig
         public uint TickRate = 30;
 
         /// <summary>
-        /// The amount of seconds to wait for handshake to complete before timing out a client
+        /// The amount of seconds for the server to wait for the connection approval handshake to complete before the client is disconnected.
+        ///
+        /// If the timeout is reached before approval is completed the client will be disconnected.
         /// </summary>
-        [Tooltip("The amount of seconds to wait for the handshake to complete before the client times out")]
+        /// <remarks>
+        /// The period begins after the <see cref="NetworkEvent.Connect"/> is received on the server.
+        /// The period ends once the server finishes processing a <see cref="ConnectionRequestMessage"/> from the client.
+        ///
+        /// This setting is independent of any Transport-level timeouts that may be in effect.
```

Comment:
```suggestion
        /// This setting is independent of any Transport-level timeouts that may be in effect. To calculate the overall connection timeout for a player connecting to a server, add transport + buffer timeout.
```

Reply:
In the new draft there are already 3-4 other references to the Transport timeout so I don't feel this explicit calculation one is necessary. It's also not necessarily accurate depending on Transport implementations.

Reply:
How about something like "a player's total connection timeout would be influenced by both transport side (if any) and network manager side timeouts". It'd just help give the big picture to users, right now we're showing this as if they were two tech in silos.